packages/jest-haste-map/src/constants.js

```diff
@@ -20,9 +20,10 @@ export default {
   /* file map attributes */
   ID: 0,
   MTIME: 1,
-  VISITED: 2,
-  DEPENDENCIES: 3,
-  SHA1: 4,
+  SIZE: 2,
```

Comment:
If you change the parameter order, be sure to force an invalidation of the Haste version to avoid reading a stale cache and crossing parameters (e.g. setting `VISITED` as `SIZE`).

Reply:
Yeah old cached haste maps are one of the things I was worried about because I was not entirely sure how the invalidation works. Looking at `getCacheFilePath`, it seems to me that the version from `jest-haste-map/package.json` is part of the cache file path hash, so when this is released as `24.0.0`, we shouldn't have a problem, right?

Reply:
Not sure what exactly you mean by forcing an invalidation though

Reply:
Uh, yeah, you're right. I thought we had a specific cache breaker just for Haste Map, but we're apparently using the version from `package.json`. Then it should be good to go!